src/main/resources/log4j.properties

```diff
@@ -0,0 +1,23 @@
+log4j.rootCategory=INFO, CONSOLE
+
+PID=????
+LOG_LEVEL_PATTERN=%5p
+LOG_PATTERN=[%d{yyyy-MM-dd HH:mm:ss.SSS}] %x boot%X{context} - ${PID} ${LOG_LEVEL_PATTERN} [%t] --- %c{1}: %m%n
+
+# CONSOLE is set to be a ConsoleAppender using a PatternLayout.
+log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
+log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
```

Comment:
To be honest, I just copied SpringBoot's default and only changed the `LOG_PATTERN` line to include the Flow-Id. :)
I'll look if it makes sense to change the PatternLayout for now. Thanks for the hint!